common/src/stack/graph_ql/app/resolvers/NetworkResolver.py

```diff
@@ -4,11 +4,59 @@
 # https://github.com/Teradata/stacki/blob/master/LICENSE.txt
 # @copyright@
 
-
+import ipaddress
 from ariadne import ObjectType, QueryType, MutationType
 import app.db as db
 
 query = QueryType()
 mutation = MutationType()
 
-object_types = []
+@query.field("networks")
+def resolve_networks(_, info, id=None, name=None):
+    query_str = "SELECT id, name, address, mask, gateway, mtu, zone, dns, pxe FROM subnets  WHERE 1 = 1"
+    if id:
+        query_str = "%s AND id = %s" % (query_str, id)
+    elif name:
+        query_str = "%s AND name = \"%s\"" % (query_str, name)
```

Comment:
I see you're carrying on MikeBob's love of SQL injections ;)

Reply:
Fixed it.